src/components/Header.jsx

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { selectUser } from '../redux/usersession/usersessionsSlice';
+import Logout from './Logout';
 
-const Header = () => (
-  <header>
-    <h1>Header</h1>
-  </header>
-);
+const Header = () => {
+  const user = useSelector(selectUser);
+  const links = [
+    { path: '/boats', text: 'Boats' },
+    { path: '/reservations', text: 'Reservations' },
```

Comment:
I kindly suggest as an optional change to update the Reservatios text to "My Reservations"